### PR TITLE
gh-108554: Changing class, methods, and variables, adding comments, fixing grammatical mistakes

### PR DESCRIPTION
**Pull request by SatinWukerORIG**
# Request to Change Names, Add Comments, and Fix Grammatical Mistakes

```
gh-108554: Summary of the changes made
```
As explained in my issue post, I made these changes to the [PDB](https://github.com/python/cpython/blob/main/Lib/pdb.py) lib:
`cre` to `compiled_pattern`
`fp` to `file_pointer`
`_rstr` to `_repr_str`
and comments:
`#run-command lines` to rcLines
`"""Inputs: frame, traceback"""` to setup(self, f, tb) due to the vagueness of `f` and `tb`
and other comments regarding the ambiguity of some code

In addition, I also linted pdb.py using `autopep8 --in-place --aggressive --aggressive pdb.py`, which fixed some misplaced comments left in the document.

Again, thank you all!👍

### CODE REVIEW

**Review by sobolevn — CHANGES_REQUESTED** [review]
Hi! Thanks a lot for your efforts, but this PR does not solve any actual issues. It introduces some extra churn, which negatively affects the project in the long run.

However, you can update you PR to just fix grammar issues (no style / renames / etc), it might be worth it.

**Comment by ericvsmith** [comment]
I agree that the vast majority of these changes are just churn and should not be accepted.

And at least one class of change, catching `BaseException`, was not mentioned in the original issue. Why is that change being proposed? What problem does it fix?

**Comment by SatinWukerORIG** [comment]
> I agree that the vast majority of these changes are just churn and should not be accepted.
> 
> And at least one class of change, catching `BaseException`, was not mentioned in the original issue. Why is that change being proposed? What problem does it fix?

_rstr() could mean regular string, raw string, representative string, etc. The same issues with fp and rcLine, which are named with ambiguity. This was the point of my pull request.
However, my apology that I used [autopep8](https://github.com/hhatto/autopep8) with aggressive fixing flag, which added the BaseException in those excepting catching lines, and I did not really check what it did semantically. I will remove the semantical changes caused by autopep8. Yet, I insist that some variables and classes' names should be modified following proper naming conventions. Thank you all for reviewing!

**Comment by SatinWukerORIG** [comment]
> > names should be modified following proper naming conventions
> 
> [Searching](https://github.com/search?q=repo%3Apython%2Fcpython+%22as+fp%22&type=code) `as fp` finds 93 results, so that seems to already be a convention (and seems proper to me).

Then it is my bad. I will change it back into `fp`. What about the other variables that are changed?

**Comment by ericvsmith** [comment]
We do not generally accept PRs that rename variables, functions, etc., or that are stylistic changes.

**Comment by sobolevn** [comment]
Please, do not change any variables :)

File: Lib/pdb.py [hunk 1]
```diff
@@ -296,6 +303,7 @@ def setup(self, f, tb):
 
     # Can be executed earlier than 'setup' if desired
     def execRcLines(self):
+        """This function executes the commands in self.rcLines; rc: run commmands"""
```
Comment (AlexWaygood) [inline Lib/pdb.py:306]: "Commands" is spelt incorrectly here

File: Lib/pdb.py [hunk 2]
```diff
@@ -92,23 +92,27 @@ class Restart(Exception):
...
-            if cre.match(line):
+    with file_pointer:
+        for lineno, line in enumerate(file_pointer, start=1):
+            if compiled_pattern.match(line):
                 return funcname, filename, lineno
     return None
 
+
 def lasti2lineno(code, lasti):
+    """Get last 2 line number"""
```
Comment (AlexWaygood) [inline Lib/pdb.py:115]: This docstring doesn't make grammatical sense

**Comment by AlexWaygood** [comment]
Thanks for the PR! However, as others have stated, our general policy is not to accept PRs that make purely stylistic changes, I'm afraid. There are many stdlib modules that do not adhere to everything PEP-8 specifies, but, as PEP-8 itself states:

> A Foolish Consistency is the Hobgoblin of Little Minds

I'm afraid I can't see any changes here that would benefit users of Python. We currently have 6,861 open issues, of which 182 have the "easy" label attached to them. I'd encourage you to take a look at one of those if you're interested in contributing to CPython :)

**Pull request closed without merging**